internal/consensus/state.go

```diff
@@ -768,6 +768,8 @@ func (cs *State) handleTimeout(
 }
 
 func (cs *State) handleTxsAvailable(ctx context.Context, stateData *StateData) {
+	// TODO: Change to trace
+	cs.logger.Debug("new transactions are available", "height", stateData.Height, "round", stateData.Round, "step", stateData.Step)
```

Comment:
You want to keep it debug for some time?

Reply:
Yes, we still have some issues with creating empty blocks too fast, need debug to investigate.